utils/rtty/files/rtty.init

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh /etc/rc.common
+
+USE_PROCD=1
+START=99
+
+BIN=/usr/sbin/rtty
+
+start_service() {
+    host=$(uci get rtty.@server[0].host)
+    port=$(uci get rtty.@server[0].port)
+    interface=$(uci -q get rtty.@device[0].interface)
+    ifname=$(uci -p /var/state get network.$interface.ifname)
+    id=$(uci -q get rtty.@device[0].id)
+    description=$(uci -q get rtty.@device[0].description)
+    ssl=$(uci -q get rtty.@server[0].ssl)
+
+    procd_open_instance
+	procd_set_param command $BIN -h $host -p $port -a
+
+    [ -n "$interface" ] && {
+        ifname=$(uci -p /var/state get network.$interface.ifname)
+        procd_append_param command -i $ifname
+    }
+
+    [ -n "$id" ] && procd_append_param command -I $id
+
+    [ "$ssl" = "1" ] && procd_append_param command -s
+
+    [ -n "$description" ] && procd_append_param command -d "$description"
+
+    procd_set_param respawn
+    procd_close_instance
+}
+
```

Comment:
Please fix the mixed indentation issue.  Tabs are preferred.

Still the config and init script combined together is not a default OFF one.  This is a service that can dig holes through layers of firewalls so it would be better if the program does not get a chance to run at all without being configured explicitly after installed.

Reply:
Fixed.

Reply:
I do not test it yet, but the current config will still cause the rtty to run with necessary arguments: `host`, `port`

Reply:
Fixed.